drivers/shared/executor/executor_test.go

```diff
@@ -77,6 +77,8 @@ func TestExecutor_Start_Invalid(pt *testing.T) {
 	pt.Parallel()
 	invalid := "/bin/foobar"
 	for name, factory := range executorFactories {
+		name := name
+		factory := factory
```

Comment:
I don't think this is necessary because the _subtest_ doesn't call t.Parallel, but either way feel free to leave it.

Reply:
Yeah I get really jumpy when I see sub tests in a for loop. There may be some historic trauma debugging a rouge t.Parallel :grimacing: